.gitignore

```diff
@@ -11,6 +11,7 @@ node_modules
 dist
 dist-ssr
 *.local
+src/generated
```

Comment:
this should not be ignored as we discussed yesterday

Reply:
Changed in 8eac28f